### ISSUE TITLE
Devcloud generated domain is inconsistent

### ISSUE THREAD

**Issue by tekante** [phase2/generator-outrigger-drupal#44]
The following comes from a `grep -r p2devcloud.com .` on a freshly generated project from the master branch. Different files (and sometimes within the same file) have different values for the assigned devcloud host. `generated/env/jenkins/config.xml` for example has a different value in the description of development than it does for QA. The deploy-ENV jobs have a different value in their descriptions than they do in the actual job script.

```
generated/build.devcloud.yml:      APP_DOMAIN: ${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/build.devcloud.yml:      GDT_DOMAIN: ${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/build.devcloud.yml:      PROXY_VIRTUAL_HOST: ${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/build.devcloud.yml:      SIMPLETEST_URL: rigconfigtest.ci2.p2devcloud.com
generated/docker-compose.yml:      VIRTUAL_HOST: db-${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/docker-compose.yml:      VIRTUAL_HOST: cache-${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/docker-compose.yml:      VIRTUAL_HOST: ${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/docker-compose.yml:      VIRTUAL_HOST: mail-${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/docker-compose.yml:      VIRTUAL_HOST: app-${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/docs/DEVCLOUD.md:All Docker-based hosted sites are located on ci.p2devcloud.com or
generated/docs/DEVCLOUD.md:ci2.p2devcloud.com.
generated/docs/DEVCLOUD.md:http://[env]-rigconfigtest.rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/config.xml:      <description>Operations for the Integration/PR Testing/Maintenance environment at &apos;int-rigconfigtest.ci2.p2devcloud.com&apos;.</description>
generated/env/jenkins/config.xml:      <description>Operations for the Development environment at &apos;dev-rigconfigtest.ci.p2devcloud.com&apos;.</description>
generated/env/jenkins/config.xml:      <description>Operations for the QA/Testing environment at &apos;qa-rigconfigtest.ci2.p2devcloud.com&apos;.</description>
generated/env/jenkins/config.xml:      <description>Operations for the Review environment at &apos;review-rigconfigtest.ci2.p2devcloud.com&apos;.</description>
generated/env/jenkins/jobs/ci/config.xml:curl -I -vvv http://${DOCKER_ENV}-rigconfigtest.ci.p2devcloud.com
generated/env/jenkins/jobs/ci/config.xml:docker-compose run --rm cli &quot;GDT_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci.p2devcloud.com grunt test --force --timer&quot;
generated/env/jenkins/jobs/deploy-dev/config.xml:  <description>Deploy changes to &apos;dev-rigconfigtest.ci.p2devcloud.com&apos;. Behat tests and code analysis are performed, user accounts seeded if configured, features health reviewed and cron is executed. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/deploy-dev/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci.p2devcloud.com
generated/env/jenkins/jobs/deploy-qa/config.xml:  <description>Deploy changes to &apos;qa-rigconfigtest.ci2.p2devcloud.com&apos;. Behat tests and code analysis are performed, user accounts seeded if configured, features health reviewed and cron is executed. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/deploy-qa/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci.p2devcloud.com
generated/env/jenkins/jobs/deploy-review/config.xml:  <description>Deploy changes to &apos;review-rigconfigtest.ci2.p2devcloud.com&apos;. Behat tests and code analysis are performed, user accounts seeded if configured, features health reviewed and cron is executed. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/deploy-review/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci.p2devcloud.com
generated/env/jenkins/jobs/dev-support/config.xml:# docker-compose run --rm cli sh -c &quot;GDT_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci.p2devcloud.com grunt test --force --timer&quot;
generated/env/jenkins/jobs/stop-dev/config.xml:  <description>Stop the &apos;dev-rigconfigtest.ci.p2devcloud.com&apos; environment. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/stop-dev/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci.p2devcloud.com
generated/env/jenkins/jobs/stop-qa/config.xml:  <description>Stop the &apos;qa-rigconfigtest.ci2.p2devcloud.com&apos; environment. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/stop-qa/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci.p2devcloud.com
generated/env/jenkins/jobs/stop-review/config.xml:  <description>Stop the &apos;review-rigconfigtest.ci2.p2devcloud.com&apos; environment. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/stop-review/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci.p2devcloud.com
generated/Gruntconfig.json:      "url": "http://backups.ci2.p2devcloud.com/rigconfigtest",
generated/jenkins.yml:      VIRTUAL_HOST: build.rigconfigtest.ci.p2devcloud.com
generated/TODOS.md:    * Visit your CI Server and [spin up your Jenkins instance](http://build.rigconfigtest.ci2.p2devcloud.com/job/ci-start/parambuild/?delay=0sec&NAME=rigconfigtest&GIT_URL=git%40bitbucket.org%3Aphase2tech%2Frigconfigtest.git&GIT_REF=develop).
```

This also appears to affect the jenkins-to-cloud branch. Here is an output from a generator run against that branch. On this particular run build.devcloud.yml and docker-compose.yml are using different hosts.
```
generated/build.devcloud.yml:      APP_DOMAIN: ${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/build.devcloud.yml:      GDT_DOMAIN: ${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/build.devcloud.yml:      PROXY_VIRTUAL_HOST: ${DOCKER_ENV:-local}-rigconfigtest.ci2.p2devcloud.com
generated/build.devcloud.yml:      SIMPLETEST_URL: rigconfigtest.ci2.p2devcloud.com
generated/docker-compose.yml:      VIRTUAL_HOST: db-${DOCKER_ENV:-local}-rigconfigtest.ci.p2devcloud.com
generated/docker-compose.yml:      VIRTUAL_HOST: cache-${DOCKER_ENV:-local}-rigconfigtest.ci.p2devcloud.com
generated/docker-compose.yml:      VIRTUAL_HOST: ${DOCKER_ENV:-local}-rigconfigtest.ci.p2devcloud.com
generated/docker-compose.yml:      VIRTUAL_HOST: mail-${DOCKER_ENV:-local}-rigconfigtest.ci.p2devcloud.com
generated/docker-compose.yml:      VIRTUAL_HOST: app-${DOCKER_ENV:-local}-rigconfigtest.ci.p2devcloud.com
generated/docs/DEVCLOUD.md:All Docker-based hosted sites are located on ci.p2devcloud.com or
generated/docs/DEVCLOUD.md:ci2.p2devcloud.com.
generated/docs/DEVCLOUD.md:http://[env]-rigconfigtest.rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/config.xml:      <description>Operations for the Integration/PR Testing/Maintenance environment at &apos;int-rigconfigtest.ci2.p2devcloud.com&apos;.</description>
generated/env/jenkins/config.xml:      <description>Operations for the Development environment at &apos;dev-rigconfigtest.ci2.p2devcloud.com&apos;.</description>
generated/env/jenkins/config.xml:      <description>Operations for the QA/Testing environment at &apos;qa-rigconfigtest.ci2.p2devcloud.com&apos;.</description>
generated/env/jenkins/config.xml:      <description>Operations for the Review environment at &apos;review-rigconfigtest.ci2.p2devcloud.com&apos;.</description>
generated/env/jenkins/jobs/ci/config.xml:curl -I -vvv http://${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/jobs/ci/config.xml:docker-compose run --rm cli &quot;GDT_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com grunt test --force --timer&quot;
generated/env/jenkins/jobs/deploy-dev/config.xml:  <description>Deploy changes to &apos;dev-rigconfigtest.ci2.p2devcloud.com&apos;. Behat tests and code analysis are performed, user accounts seeded if configured, features health reviewed and cron is executed. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/deploy-dev/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/jobs/deploy-qa/config.xml:  <description>Deploy changes to &apos;qa-rigconfigtest.ci2.p2devcloud.com&apos;. Behat tests and code analysis are performed, user accounts seeded if configured, features health reviewed and cron is executed. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/deploy-qa/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/jobs/deploy-review/config.xml:  <description>Deploy changes to &apos;review-rigconfigtest.ci2.p2devcloud.com&apos;. Behat tests and code analysis are performed, user accounts seeded if configured, features health reviewed and cron is executed. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/deploy-review/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/jobs/dev-support/config.xml:# docker-compose run --rm cli sh -c &quot;GDT_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com grunt test --force --timer&quot;
generated/env/jenkins/jobs/start-dev/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/jobs/start-qa/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/jobs/start-review/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/jobs/stop-dev/config.xml:  <description>Stop the &apos;dev-rigconfigtest.ci2.p2devcloud.com&apos; environment. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/stop-dev/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/jobs/stop-qa/config.xml:  <description>Stop the &apos;qa-rigconfigtest.ci2.p2devcloud.com&apos; environment. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/stop-qa/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com
generated/env/jenkins/jobs/stop-review/config.xml:  <description>Stop the &apos;review-rigconfigtest.ci2.p2devcloud.com&apos; environment. This job is managed in the project code repository, changes made via UI will be lost.</description>
generated/env/jenkins/jobs/stop-review/config.xml:export SITE_DOMAIN=${DOCKER_ENV}-rigconfigtest.ci2.p2devcloud.com
generated/Gruntconfig.json:      "url": "http://backups.ci2.p2devcloud.com/rigconfigtest",
generated/jenkins.yml:      VIRTUAL_HOST: build.rigconfigtest.ci2.p2devcloud.com
generated/TODOS.md:    * Visit your CI Server and [spin up your Jenkins instance](http://build.rigconfigtest.ci.p2devcloud.com/job/ci-start/parambuild/?delay=0sec&NAME=rigconfigtest&GIT_URL=git%40bitbucket.org%3Aphase2tech%2Frigconfigtest.git&GIT_REF=develop).
```

**Comment by tekante** [comment]
Would be implicitly fixed by #52

**Comment by grayside** [comment]
Indeed. I've tested a generated project with all the bells and whistles using the grep command in the OP and it all looks good.
